code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -230,7 +230,7 @@
 
 /mob/living/carbon/human/equipped_speed_mods()
 	. = ..()
-	for(var/sloties in get_all_worn_items() - list(l_store, r_store, s_store))
+	for(var/sloties in get_all_worn_items() - list(l_store, r_store, s_store, back, wear_mask, wear_neck, head, handcuffed, legcuffed))
```

Comment:
thonk

Reply:
Wasn't sure how to get carbon's get_all_worn_items() from within the proc, so I did what I had to do to make it work.

Reply:
thonker.